util/service_requests.py

```diff
@@ -22,8 +22,8 @@ def get_call_or_raise(url: str, params: Dict = {}) -> Any:
         raise GraphQLError(response.text)
 
 
-def delete_call_or_raise(url: str) -> int:
-    response = requests.delete(url=url)
+def delete_call_or_raise(url: str, params: Dict = {}) -> int:
```

Comment:
Dont use a Dict as a default value (see [notion](https://www.notion.so/kern-ai/Coding-Guide-Python-f54c6201aa22471caaff38d765c0d64e#8ea49ab353b54cb3960cadafb4307389) for reference)